manager/resolver/maven/src/main/java/org/jboss/forge/furnace/manager/maven/addon/MavenAddonDependencyResolver.java

```diff
@@ -226,6 +285,36 @@ public DependencySelector deriveChildSelector(DependencyCollectionContext contex
       return new MavenResponseBuilder<String>(apiVersion).setExceptions(exceptions);
    }
 
+   private ArtifactDescriptorResult readArtifactDescriptor(AddonId addonId) throws ArtifactDescriptorException
+   {
+      String coords = toMavenCoords(addonId);
+      RepositorySystem system = container.getRepositorySystem();
+      Settings settings = getSettings();
+      DefaultRepositorySystemSession session = container.setupRepoSession(system, settings);
+      List<RemoteRepository> repositories = MavenRepositories.getRemoteRepositories(container, settings);
+      ArtifactDescriptorRequest request = new ArtifactDescriptorRequest();
+      request.setArtifact(new DefaultArtifact(coords));
+      request.setRepositories(repositories);
+
+      ArtifactDescriptorResult result = system.readArtifactDescriptor(session, request);
```

Comment:
I think that this is also implemented in MavenDependencyResolver#getArtifactMetadata() but I'm not sure we can re-use that.